GoogleForCreators/web-stories-wp
# Dashboard: Fixed button placement and swapped watch in Fitness template

## Summary

This PR makes some quick fixes to the Fitness template.

The following updates were made:

1.) The "Buy Now" button and above text was moved towards the center so they're not right up against the edge:
<img src="https://user-images.githubusercontent.com/40646372/86175604-811bb380-bad8-11ea-9f12-6b12747d3f1a.png" height="400">

2.) Swapped out the Apple watch with a fitbit:
<img src="https://user-images.githubusercontent.com/40646372/86175658-998bce00-bad8-11ea-98da-1cf6c2d2f3dc.png" height="400">

## User-facing changes

- Small text changes.

## Testing Instructions

1.) Go to "Explore Templates"
2.) Click on "View" for "Fitness" template and check that pages 5 and 6 appear as the images above.

Fixes #2473
